### PR TITLE
topolvm.io

### PR DESCRIPTION
**Pull request by bells17**
This is a merge PR for reducing the number of commits for #539

- #539
- #168
- #589

The below commands created this PR:

```
$ git checkout main
$ git pull
$ git checkout topolvm.io
$ git merge main
$ git push
$ git diff main > /tmp/topolvm.io.patch
$ git checkout main
$ git co -b topolvm.io4
$ git apply /tmp/topolvm.io.patch
$ git add .
$ git commit -m "topolvm.io"
$ git push
```

### CODE REVIEW

File: charts/topolvm/README.md.gotmpl [hunk 1]
```diff
@@ -88,6 +88,13 @@ You can use the `helm template` command to render manifests.
 helm template --include-crds --namespace=topolvm-system topolvm topolvm/topolvm
 ```
 
+## About the legacy flag
+
+In https://github.com/topolvm/topolvm/pull/539, the domain name which is used for CRD and other purposes was changed from topolvm.cybozu.com to topolvm.io.
```
Comment (bells17) [inline charts/topolvm/README.md.gotmpl:93]: https://github.com/topolvm/topolvm/pull/589#discussion_r1017756412

File: charts/topolvm/README.md.gotmpl [hunk 2]
```diff
@@ -90,7 +90,7 @@ helm template --include-crds --namespace=topolvm-system topolvm topolvm/topolvm
 
 ## About the legacy flag
 
-In https://github.com/topolvm/topolvm/pull/539, the domain name which is used for CRD and other purposes was changed from topolvm.cybozu.com to topolvm.io.
+In https://github.com/topolvm/topolvm/pull/592, the domain name which is used for CRD and other purposes was changed from topolvm.cybozu.com to topolvm.io.
```
Comment (bells17) [inline charts/topolvm/README.md.gotmpl:93]: https://github.com/topolvm/topolvm/pull/589#discussion_r1017756412

**Review by nbalacha — DISMISSED** [review]

File: docs/maintenance.md [hunk 3]
```diff
@@ -31,6 +31,7 @@ Edit the following files.
 - `Makefile`
 - `e2e/Makefile`
 - `example/Makefile`
+- `example/migrate/kube-scheduler.yaml`
```
Comment (peng225) [inline docs/maintenance.md:34]: This file was removed, so please delete this line.

**Review by peng225 — APPROVED** [review]

File: docs/maintenance.md (hunk 3 above)
Comment (peng225) [inline docs/maintenance.md:34]: `example/migrate/kube-scheduler.yaml` was removed, so please delete this line.

**Comment by nbalacha** [comment]
What I usually do when I need to change something in a commit is the following:
git reset --soft HEAD~<num-commits>
Make the changes and git add
git commit again
git push -u origin <branch name> --force

This will overwrite the branch head so I can use the same PR. It may be a way to proceed with these changes instead of creating a new PR.

File: docs/maintenance.md (hunk 3 above)
Comment (bells17) [reply docs/maintenance.md]: Sorry, I deleted it.

https://github.com/topolvm/topolvm/pull/592/commits/b7a6c14de6957c9dcf8e60c56161e7e41eca6438

**Comment by bells17** [comment]
> What I usually do when I need to change something in a commit is the following:

Thank you. I never tried the `git reset --soft` command.
I'll try these commands the next time.

**Review by llamerada-jp — COMMENTED** [review]
This PR looks good. Would you please squash this PR after other reviewers approve?

**Comment by peng225** [comment]
LGTM. I will approve it after you squashed it.

**Review by llamerada-jp — APPROVED** [review]

**Review by peng225 — APPROVED** [review]

**Pull request merged**